build_common.ps1

```diff
@@ -8,6 +8,11 @@ $global:buildCommonSelfPath = split-path -parent $MyInvocation.MyCommand.Definit
 # list of all native script packages
 $global:nativescriptpackages = @("XComGame", "Core", "Engine", "GFxUI", "AkAudio", "GameFramework", "UnrealEd", "GFxUIEditor", "IpDrv", "OnlineSubsystemPC", "OnlineSubsystemLive", "OnlineSubsystemSteamworks", "OnlineSubsystemPSN")
 
+# Enforce decimal separator dot. Sorry.
+$culture = [System.Globalization.CultureInfo]::CreateSpecificCulture("en-US")
+[System.Threading.Thread]::CurrentThread.CurrentCulture = $culture
```

Comment:
This could potentially break ModBuddy, since the powershell script is executed in-process (not in a child/standalone process like in a command line invocation)